src/nomad_simulations/schema_packages/model_method.py

```diff
@@ -1221,3 +1221,259 @@ class DMFT(ModelMethodElectronic):
 
     def normalize(self, archive: 'EntryArchive', logger: 'BoundLogger') -> None:
         super().normalize(archive, logger)
+
+
+class MolecularOrbital(ArchiveSection):
+    """
+    A section representing a single molecular orbital.
+    """
+
+    energy = Quantity(
+        type=np.float64,
+        # unit='electron_volt',
+        description='Energy of the molecular orbital.',
+    )
+
+    occupation = Quantity(
+        type=np.float64, description='Occupation number of the molecular orbital.'
+    )
+
+    symmetry_label = Quantity(
+        type=str, description='Symmetry label of the molecular orbital.'
+    )
+
+    reference_orbital = SubSection(
+        sub_section=OrbitalsState.m_def,
+        description='Reference to the underlying atomic orbital state.',
+    )
```

Comment:
What about `AtomicState`?

Reply:
If you don't use a common super class, you'd have to add some normalization here too.